ipxe/custom.go

```diff
@@ -0,0 +1,23 @@
+package ipxe
+
+import "net/url"
+
+// CustomScript is the template for the custom script.
+// It will either chain to a URL or execute an iPXE script.
+var CustomScript = `#!ipxe
+
+echo Loading the Tinkerbell iPXE custom script...
```

Comment:
Would this read better as the following?

```suggestion
echo Loading custom Tinkerbell iPXE script ...
```

Reply:
sure, updated.